docs/aeso_compiler.md

```diff
@@ -51,6 +51,32 @@ The **pp_** options all print to standard output the following:
 
 `pp_assembler` - print the generated assembler code
 
+#### Options to control which compiler optimizations should run:
+
+By default all optimizations are turned on, to disable an optimization, it should be
+explicitly set to false and passed as a compiler option.
+
+List of optimizations:
+
+##### optimize_inliner
+##### optimize_inline_local_functions
+##### optimize_bind_subexpressions
+##### optimize_let_floating
+##### optimize_simplifier
+##### optimize_drop_unused_lets
+##### optimize_push_consume
+##### optimize_one_shot_var
+##### optimize_write_to_dead_var
+##### optimize_inline_switch_target
+##### optimize_swap_push
+##### optimize_swap_pop
+##### optimize_swap_write
+##### optimize_constant_propagation
+##### optimize_prune_impossible_branches
+##### optimize_single_successful_branch
+##### optimize_inline_store
+##### optimize_float_switch_bod
```

Comment:
Either make use of the sections by providing some descriptions or make it, or just use a list
```suggestion
- optimize_inliner
- optimize_inline_local_functions
- optimize_bind_subexpressions
- optimize_let_floating
- optimize_simplifier
- optimize_drop_unused_lets
- optimize_push_consume
- optimize_one_shot_var
- optimize_write_to_dead_var
- optimize_inline_switch_target
- optimize_swap_push
- optimize_swap_pop
- optimize_swap_write
- optimize_constant_propagation
- optimize_prune_impossible_branches
- optimize_single_successful_branch
- optimize_inline_store
- optimize_float_switch_bod
```